### PR TITLE
New Crowdin translations

### PR DESCRIPTION
**Pull request by nicarq**
Enjoy!

### CODE REVIEW

File: src/l10n/jp-JP.js [hunk 1]
```diff
@@ -157,7 +156,9 @@ const l10n = {
   MnemonicExplanationModal: {
     paragraph1: [
       inline([
-        normal('次の画面で、任意の15個の言葉が表示されます。 '),
+        normal(
+          '次の画面で、任意の15個の言葉が表示されます。 ',
```
Comment (nicarq) [inline src/l10n/jp-JP.js]: @Leo-Emurgo I think there is an extra space on this one

File: src/l10n/jp-JP.js [hunk 2]
```diff
@@ -185,7 +186,7 @@ const l10n = {
     repeatPasswordInput: {
       label: 'パスワードをもう一度入力してください。',
       errors: {
-        passwordsDoNotMatch: 'パスワードが間違っています。',
+        passwordsDoNotMatch: '2つのパスワードが違います。',
```
Comment (nicarq) [inline src/l10n/jp-JP.js]: @Leo-Emurgo is this 2 in the beginning on purpose?

File: src/l10n/jp-JP.js [hunk 3]
```diff
@@ -487,11 +493,10 @@ const l10n = {
       paragraph1:
         'ウォレットを完全に消去したい場合は、' +
         '復元フレーズを保持していることを確認してください。',
-      paragraph2:
-        '操作を承認するために、ウォレット名を下にタイプしてください。',
+      paragraph2: '操作を承認するために、ウォレット名を下にタイプしてください。',
     },
     walletName: 'ウォレット名',
-    walletNameInput: 'ウォレット名',
+    walletNameInput: 'ウォレット名'
```
Comment (nicarq) [inline src/l10n/jp-JP.js]: @Leo-Emurgo I think we are missing a comma at the end

File: src/l10n/jp-JP.js [hunk 4]
```diff
@@ -521,7 +526,7 @@ const l10n = {
     repeatPasswordInput: {
       label: '新しいパスワードをもう一度入力してください。',
       errors: {
-        passwordsDoNotMatch: 'パスワードが間違っています。',
+        passwordsDoNotMatch: '２つのパスワードが違います',
```
Comment (nicarq) [inline src/l10n/jp-JP.js]: @Leo-Emurgo is the 2 on purpose here as well?

File: src/l10n/jp-JP.js [hunk 5]
```diff
@@ -548,7 +553,9 @@ const l10n = {
       heading:
         '簡易トランザクション承認とは、指紋認証や顔認証 ' +
         'を用いて、ADAのトランザクションを行うものです。 ' +
+        ' ' +
```
Comment (nicarq) [inline src/l10n/jp-JP.js]: @Leo-Emurgo can we delete this part plz and also the extra line two lines below 🙏

File: src/l10n/jp-JP.js (hunk 2 above)
Comment (Leo-Emurgo) [reply src/l10n/jp-JP.js]: Yes. I will be making a lot of edits this afternoon, though. So please don't apply any changes until tonight :)

File: src/l10n/jp-JP.js (hunk 1 above)
Comment (nicarq) [reply src/l10n/jp-JP.js]: oh it’s a bug from Crowdin, there is definitely an space there

**Pull request merged**